packages/core/src/components/BaseInput/BaseInput.module.scss

```diff
@@ -0,0 +1,118 @@
+@import "../../styles/Typography";
+
+.wrapper {
+  width: 100%;
+  position: relative;
+  display: flex;
+  align-items: center;
+  gap: var(--spacing-small);
+  padding-block: var(--spacing-xs);
+  padding-inline: var(--spacing-medium) var(--spacing-xs);
+
+  @include vibe-text(text1, normal);
+  @include smoothing-text;
+
+  outline: none;
+  border: 1px solid var(--ui-border-color);
+  border-radius: var(--border-radius-small);
+  color: var(--primary-text-color);
+  background-color: var(--secondary-background-color);
+  transition: border-color var(--motion-productive-medium) ease-in;
+
+  &.small {
+    height: 32px;
+    @include vibe-text(text2, normal);
+  }
+
+  &.medium {
+    height: 40px;
+  }
+
+  &.large {
+    height: 48px;
+    padding-block: var(--spacing-small);
+  }
+
+  &.underline {
+    border-left: none;
+    border-right: none;
+    border-top: none;
+    padding: var(--spacing-small) var(--spacing-large) var(--spacing-small) var(--spacing-xs);
+    border-radius: 0 !important;
```

Comment:
Necesarry?

Reply:
The PR was still in draft state, copied some of the logic from `TextField`, removed unnecessary stuff now